atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTableTest.java

```diff
@@ -308,6 +311,30 @@ public void createDoesNotThrowTableMappingExceptionWhenTableIsNotListedToBeAlert
         assertThatCode(() -> ddlTable.create(createMetadata(true))).doesNotThrowAnyException();
     }
 
+    @Test
+    public void createDoesNotThrowWhenOverflowColumnAlreadyExistsAndAlterIsRan() throws TableMappingNotFoundException {
+        createTableAndOverflow();
+        setTableToHaveOverflowColumn(false);
+        setTableValueStyleCacheOverflowConfigForTable(true);
+        doThrow(PalantirSqlException.create(OracleErrorConstants.ORACLE_COLUMN_ALREADY_EXISTS))
+                .when(sqlConnection)
+                .executeUnregisteredQuery("ALTER TABLE " + INTERNAL_TABLE_NAME + " ADD (overflow NUMBER(38))");
+        tableMappingDdlTable.create(createMetadata(true));
+        verifyTableAltered();
+    }
+
+    private void testAlterTableForMigrationState(OverflowMigrationState overflowMigrationState)
```

Comment:
it's a shame we can't easily have nice `@ParameterizedTest` (without pulling extra packages)- you could use the class level one, but I don't think there's much value there. I'd recommend just putting this method directly underneath `createDoesNothingWhenAlterSpecifiedButMigrationFinishing` so it's closer to the relevant tests